adapter/datadog/logger_test.go

```diff
@@ -0,0 +1,30 @@
+package datadog
+
+import (
+	"testing"
+
+	coopLogger "github.com/coopnorge/go-logger"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace"
+)
+
+func TestSetup(_ *testing.T) {
+	logger, _ := NewLogger(WithGlobalLogger()) // nolint:errcheck
```

Comment:
Same comments as in the Gorm-adapter: I prefer to use stretchr/testify/require instead of `nolint`

Reply:
Errors are now checked